packages/repository-modernizer/src/restructure-pom.js

```diff
@@ -148,6 +148,22 @@ var RestructurePoms = {
                 pluginObj,
                 conversionStep
             );
+            if (config.parentPom.path == null) {
+                let parentPomarr = [
```

Comment:
`parentPomInfo` would be better name. Also an object with named properties would be better than an array here.

`{ path : project.parentPom.path, artifactId : project.parentPom.artifactId, appTitle : project.parentPom.appTitle }`